### PR TITLE
Text component

### PR DESCRIPTION
**Pull request by alycrys**

### CODE REVIEW

File: packages/core/src/text/Display.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { makePrefixer } from "@jpmorganchase/uitk-core";
```
Comment (joshwooding) [inline packages/core/src/text/Display.tsx:1]:
```suggestion
import { makePrefixer } from "../utils";
```

Comment (alycrys) [reply packages/core/src/text/Display.tsx]: Thank you! It was in front of my eyes, I just couldn't see it

File: packages/lab/stories/link.stories.tsx [hunk 2]
```diff
@@ -51,7 +51,10 @@ export const TargetBlankNoIcon: ComponentStory<typeof Link> = () => {
 export const WithTooltip: ComponentStory<typeof Link> = () => {
```
Comment (joshwooding) [inline packages/lab/stories/link.stories.tsx:51]: We should hide this story

File: packages/theme/css/theme.css [hunk 3]
```diff
@@ -35,3 +35,8 @@
 @import url(characteristics/status.css);
 @import url(characteristics/taggable.css);
 @import url(characteristics/text.css);
+
```
Comment (joshwooding) [inline packages/theme/css/theme.css:38]: Just curious, why was this put here instead of in global?

Comment (joshwooding) [reply packages/theme/css/theme.css:38]: Global is included when you import the theme anyway

**Comment by joshwooding** [comment]
Awesome work @alycrys 🎉

File: packages/theme/css/theme.css (hunk 3 above)
Comment (alycrys) [reply packages/theme/css/theme.css]: There was a discussion about having it as part of theme, but I think the point was more like having (maybe) some global styling per theme. Not sure if that's something we want to do. But either way it should be done in a different PR.

**Review by joshwooding — APPROVED** [review]

**Comment by sion-docs** [comment]
The template needs to be applied to this documentation - the following sections/headings are missing:
- Configuring Text
- Accessibility
- Help and support

**Pull request merged**